go.mod

```diff
@@ -88,3 +89,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/kubeflow/common v0.4.5 => github.com/tenzen-y/common v0.0.0-20230115192234-865b54409a0b
```

Comment:
After https://github.com/kubeflow/common/pull/203 is merged, I must remove this replacement.

Reply:
After creating a new release for kubeflow/common, I will remove this replacement.